src/log.c

```diff
@@ -32,6 +67,12 @@ static void dunst_log_handler (
         if(testing)
                 return;
 
+/* if you want to have a debug build, you want to log anything,
+ * unconditionally, without specifying debug log level again */
+#ifndef DEBUG_BUILD
+        if(log_level > message_level)
```

Comment:
This needs to be less than, not greater than. It currently accomplishes the exact opposite thing of what it's supposed to do.

```
(gdb) p (int)G_LOG_LEVEL_CRITICAL
$20 = 8
(gdb) p (int)G_LOG_LEVEL_WARNING
$21 = 16
```